app/src/main/java/android/app/lotus/view/account/SupportView.kt

```diff
@@ -22,10 +22,13 @@ import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import androidx.lifecycle.ViewModel
 
+class SupportViewModel : ViewModel(){
+    var message: String by mutableStateOf("")
+}
 @Composable
-fun SupportView(){
-    var message by remember { mutableStateOf("") }
+fun SupportView(viewModel: SupportViewModel){
```

Comment:
ViewModel should not be passed to this view. Instantiate the viewmodel inside the composable with hiltViewModel() after fixing the above comment.